### PR TITLE
qemu_guest_agent: check agent status when no user logged in

### PR DESCRIPTION
**Pull request by xiagao**
new case of cheing guest agent status when no user login,only
for windows guest.
ID: 1761658
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
(1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.gagent_user_logoff: PASS (88.81 s)

**Comment by xiagao** [comment]
@xiangchunfu  call for your review, thanks.

File: qemu/tests/qemu_guest_agent.py [hunk 1]
```diff
@@ -1949,6 +1949,34 @@ def gagent_check_with_migrate(self, test, params, env):
...
+        session = self._get_session(params, None)
+        self._open_session_list.append(session)
+
+        error_context.context("Check which user is logged in.", logging.info)
+        user_info = session.cmd_output('query user | findstr /i "Active"')
+        login_user_id = user_info.strip().split()[2]
+
+        error_context.context("Make the user log out.", logging.info)
+        try:
+            session.cmd("logoff %s" % login_user_id)
```
Comment (xiangchunfu) [inline qemu/tests/qemu_guest_agent.py]: @xiagao This case don't need to cover windows guest,right?

Comment (xiagao) [reply qemu/tests/qemu_guest_agent.py]: > @xiagao This case don't need to cover windows guest,right?

it's only cover windows guest. thanks.

**Comment by xiangchunfu** [comment]
Acked by: Xiangchun Fu xfu@redhat.com

**Comment by xiagao** [comment]
@lolyu  Could you help to review this patch? Thanks.

**Comment by xiagao** [comment]
(1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.gagent_user_logoff: PASS (108.08 s)
@lolyu Thanks.

**Pull request merged**